This patch is for festival for Debian GNU/Linux, to add text2wave
man page, ensure man pages pass lintian hyphen test and are built
from correct directories for Debian filesystem layout

Index: b/doc/Makefile
===================================================================
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -44,7 +44,7 @@
 
 EXAMPLE_TO_DOCUMENT=
 
-FESTIVAL=$(TOP)/bin/festival --libdir $(TOP)/lib
+FESTIVAL=$(TOP)/src/main/festival --libdir $(TOP)/lib --datadir $(TOP)/lib
 
 # Include some of EST documentation.
 DOCXX_EXTRA_FILES = 
@@ -56,7 +56,8 @@
 	     banner.inc $(DOCXXIMAGES)
 
 MANPAGES = festival.head festival.tail \
-           festival_client.head festival_client.tail
+           festival_client.head festival_client.tail \
+		   text2wave.head text2wave.tail
 
 #SGMLFILES = festival.sgml \
 #	introductory.sgml basics.sgml core.sgml advanced.sgml programming.sgml
@@ -64,7 +65,7 @@
 FILES=Makefile $(MANPAGES) festival.texi $(SGMLFILES) refcard.tex  $(DOCXXFILES)
 LOCAL_CLEAN = *.aux *.cp *.fn *.ky *.log *.pg *.toc *.tp *.vr
 
-ALL = festival.1 festival_client.1
+ALL = festival.1 festival_client.1 text2wave.1
 
 include $(TOP)/config/common_make_rules
 include $(EST)/config/rules/doc.mak
@@ -72,9 +73,9 @@
 %.1 : %.head %.options %.tail 
 	cat $^ >$@
 %.options : $(TOP)/src/main/%
-	$(TOP)/bin/$* -man_options >$@
+	$(TOP)/src/main/$* -man_options |sed 's#--#\\-\\-#' >$@
 %.options : $(TOP)/src/main/%.exe
-	$(TOP)/bin/$* -man_options >$@
+	$(TOP)/src/main/$* -man_options >$@
 
 festival.info: festival.texi festfunc.texi festvars.texi festfeat.texi
 	@ if [ ! -d info ] ; \
@@ -104,8 +105,8 @@
 	tex festival.texi
 doc:	festival.ps festival.html festival.info
 
-festfunc.texi festvars.texi festfeat.texi: $(TOP)/src/main/festival
-	echo "(load_library \"festdoc.scm\") (make-doc)" | $(FESTIVAL)
+festfunc.texi festvars.texi festfeat.texi: $(TOP)/src/main/festival doc.scm
+	$(FESTIVAL) < doc.scm
 
 refcard.dvi:	refcard.tex
 	latex refcard.tex
